Configure

```diff
@@ -1227,6 +1227,9 @@ foreach my $what (sort keys %disabled) {
         $skipdir = "ripemd" if $what eq "rmd160";
         $skipdir = "whrlpool" if $what eq "whirlpool";
 
+        # no-cmp also disables building of crmf
+        $skipdir{"crypto/crmf"} = $what if $what eq "cmp";
```

Comment:
I thought that #8897 handles the cmp/crmf disable stuff?

Reply:
Right, this is (meanwhile at least) needless; removed.